database/migrations/2020_06_17_064623_add_uuid_to_users_table.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddUuidToUsersTable extends Migration
```

Comment:
All existing migrations are anonymous, let's make the new migrations anonymous too